### ISSUE TITLE
Google adsense

### ISSUE THREAD

**Issue by cpeuschel** [marcantondahmen/automad#67]
Hi,

is there any kind of adsense plugin?

**Comment by locdown2311** [comment]
I don't think so. Add to the header and create an HTML block with your ad code. I believe it should work, something like that
